packages/ui/src/utils/exceptions/hasPNCException.ts

```diff
@@ -0,0 +1,15 @@
+import type { Exception } from "types/exceptions"
+import ExceptionCode from "@moj-bichard7-developers/bichard7-next-data/dist/types/ExceptionCode"
+
+const PNCExceptions = [ExceptionCode.HO100310, ExceptionCode.HO100332]
```

Comment:
`ExceptionCode.HO100310, ExceptionCode.HO100332` are offence matching exceptions. So you rename your function and filename.

PNC Exceptions are something else like https://dsdmoj.atlassian.net/wiki/spaces/KB/pages/4612554864/Exceptions#HO100304


We don't need to make a new array see: 
`packages/ui/src/utils/offenceMatcher/offenceMatchingExceptions.ts`